tests/functional_tests/mining.py

```diff
@@ -170,6 +232,11 @@ def submitblock(self):
             res = daemon.get_height()
             assert res.height == height + i + 1
             assert res.hash == block_hash
+            
+    def printTimeTaken(self, start, msgContext):
```

Comment:
Why is this using camelcase when everything else follows the standard python convention of lowercase, underscore separated identifiers?

Reply:
Overlook

Reply:
Done.